### PR TITLE
REP-6654 Docs updates.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+== General Service information
+* *Default Configuration:* open-tracing.cfg.xml
+* *Released:* vTBD
+* link:../schemas/open-tracing.xsd[Schema]
+
+== Tracer implementation detail
+
+This section will cover all Tracers that are supported by the OpenTracing service.
+We will be adding to this list as more tracers as supported.
```
Comment (adrianjgeorge) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: "more tracers **are** supported."

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 2]
```diff
@@ -0,0 +1,133 @@
+= OpenTracing Service
+
+The OpenTracing service (http://opentracing.io/) is a service in *Repose* which allows for sending tracing data to a Tracing service.
```
Comment (dmnjohns) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Something about the way this link is presented bothers me ever so slightly. I propose something like this:
> The OpenTracing service, which adheres to the http://opentracing.io[OpenTracing standard], enables reporting of tracing data to a collection service.

Of course, that's just my preference, so feel free to use some, all, or none of that.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+The initial release of the OpenTracing service only supports using the Jaeger (http://jaeger.readthedocs.io/en/latest/) Tracer.
```
Comment (dmnjohns) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Perhaps:

> ... using the http://jaeger.readthedocs.io/en/latest/[Jaeger client library].

After all, the `Tracer` is just one component in that library, even if it is our entrypoint into pretty much everything else.

Then for the future line, you might say other libraries or other OpenTracing implementations.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+However, other Tracers can be integrated in future OpenTracing service updates.
+By default, the service is disabled, but it can be enabled by including an `open-tracing.cfg.xml` configuration.
```
Comment (dmnjohns) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Do we want to make a note that the service cannot be disabled once it has been enabled unless Repose is restarted?

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+The origin service can simply pass the Tracer-specific header to downstream services and requests will be traced across application boundaries.
+
+== General Service information
+* *Default Configuration:* open-tracing.cfg.xml
+* *Released:* vTBD
+* link:../schemas/open-tracing.xsd[Schema]
+
+== Tracer implementation detail
```
Comment (dmnjohns) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Also maybe call this the `OpenTracing Implementation Details` or something if you agree with the reasons I gave for the previous change .

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+|Tracer Name |Tracer Header    |Extraction examples
+
+|*Jaeger*
+|`uber-trace-id`
+|https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-apachehttpclient
+|===
+
+== Examples
+=== Basic Configuration
+This configuration will send all traces to the default HTTP collector at `http://localhost:14268`.
```
Comment (dmnjohns) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Just so we don't forget, the default endpoint has been dropped.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+* link:../schemas/open-tracing.xsd[Schema]
+
+== Tracer implementation detail
+
+This section will cover all Tracers that are supported by the OpenTracing service.
+We will be adding to this list as more tracers as supported.
+
+Current support:
+
+.Table Tracer information
```
Comment (dmnjohns) [inline repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Seems weird to label the Table as Table? I feel like `Tracer information` would suffice, but eh. Or some other name if we aren't calling everything a `Tracer`.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 1 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Updated.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 2 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Updated.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 3 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Updated.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 4 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Added.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 5 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Updated.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 6 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Updated.

File: repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc (hunk 7 above)
Comment (wdschei) [reply repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc]: Updated.

**Pull request merged**